pkg/kubectl/cmd/label.go

```diff
@@ -103,7 +104,7 @@ func parseLabels(spec []string) (map[string]string, []string, error) {
 	for _, labelSpec := range spec {
 		if strings.Index(labelSpec, "=") != -1 {
 			parts := strings.Split(labelSpec, "=")
-			if len(parts) != 2 {
+			if len(parts) != 2 || len(parts[1]) == 0 || !util.IsValidLabelValue(parts[1]) {
 				return nil, nil, fmt.Errorf("invalid label spec: %v", labelSpec)
```

Comment:
I know it's preexisting, but invalid user input should return [`UsageError`](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/util/helpers.go#L163). Do you mind fixing references in this file?